src/tls13.c

```diff
@@ -10628,7 +10628,22 @@ int wolfSSL_send_hrr_cookie(WOLFSSL* ssl, const unsigned char* secret,
 
     return ret;
 }
-#endif
+
+void wolfSSL_disable_hrr_cookie(WOLFSSL * ssl)
+{
+    if (ssl->buffers.tls13CookieSecret.buffer != NULL) {
```

Comment:
Public function needs to check `ssl` for NULL. Does this apply to both the client and server?

Reply:
Nice point!
Changed to check that `ssl != NULL`, that the function is invoked on the right side and that it always errors out if `NO_WOLFSSL_SERVER` is defined. Changing return value to int to allow returning errors.